src/main/java/rocks/inspectit/gepard/agent/transformation/advice/InspectitAdvice.java

```diff
@@ -8,16 +8,17 @@
 public class InspectitAdvice {
 
   @Advice.OnMethodEnter(suppress = Throwable.class)
-  public static void onEnter(@Advice.AllArguments Object[] args, @Advice.This Object thiz) {
-    System.out.println("HELLO GEPARD");
+  public static void onEnter(@Advice.AllArguments Object[] args, @Advice.This Object thiz, @Advice.Origin String method) {
+    System.out.println("HELLO GEPARD from: " + thiz.getClass().getName() + " in method: " + method);
```

Comment:
Can you move `thiz.getClass().getName()` into a separate line, to keep it cleaner.